src/components/sbb-link/sbb-link.tsx

```diff
@@ -160,21 +176,22 @@ export class SbbLink implements LinkButtonProperties, ComponentInterface {
 
   public render(): JSX.Element {
     let TAG_NAME: string;
-    let attributeList: object = getLinkButtonBaseAttributeList(
-      this.idValue,
-      this._getClassString(),
-      this
-    );
+    let attributeList: object = getLinkButtonBaseAttributeList(this);
     if (this._isStatic) {
       TAG_NAME = 'span';
     } else {
       TAG_NAME = this.href ? 'a' : 'button';
-      attributeList = getLinkButtonAttributeList(this.idValue, this._getClassString(), this);
+      attributeList = getLinkButtonAttributeList(this);
     }
 
     // See https://github.com/ionic-team/stencil/issues/2703#issuecomment-1050943715 on why form attribute is set with `setAttribute`
     return (
-      <TAG_NAME {...attributeList} ref={(btn) => this.form && btn?.setAttribute('form', this.form)}>
+      <TAG_NAME
+        {...(this.idValue && { id: this.idValue })}
```

Comment:
I'm not sure I like this usage. Could we not just use `id={this.idValue}`?
I assume jsx will validate this anyway.